Linguini.Bundle.Test/Linguini.Bundle.Test.csproj

```diff
@@ -24,4 +24,9 @@
         <AdditionalFiles Include="..\PluralRules.Generator\cldr_data\plurals.xml" />
     </ItemGroup>
 
+    <ItemGroup Condition="'$(TargetFramework)' != 'net5.0'">
```

Comment:
Some of these use `'$(TargetFramework)' != 'net5.0'`, and some use `'$(TargetFramework)' == 'net461'`. Was that intentional?